api.md

```diff
@@ -170,6 +171,118 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() [](add_data_node)
+
+Add a new data node to the current database for use with distributed
```

Comment:
Could we add a sentence here that says something along the lines of - this command must be run on an access node?

Reply:
Done.